tensorflow/stream_executor/stream.cc

```diff
@@ -5242,16 +5242,19 @@ Stream &Stream::ThenCtcLoss(const dnn::RnnStateTensorDescriptor &probs_desc,
   if (ok()) {
     if (dnn::DnnSupport *dnn = parent_->AsDnn()) {
       DeviceMemory<uint8> scratch_memory;
+      int ctc_loss_algo_id = 0;
```

Comment:
I'd suggest having `PrepareForCtcLoss` always initialize this on success then leave it uninitialized here.